Core/StartupArgManager.cs

```diff
@@ -37,5 +36,27 @@ public static string[] MergeStartupArgs(IEnumerable<string> cmdArgs)
                 return startupArgs.ToArray();
             }
         }
+
+        public static void MigrateArgsFile()
+        {
+            try
+            {
+                string OldFileLocation = "laargs.txt";
```

Comment:
1. This should be a const.
2. The same string is used in Program.cs as well. Preferably the file check should be made in this method and it can be called from MergeStartupArgs.

Reply:
Updated the location for the file check.